README.md

```diff
@@ -56,6 +56,12 @@ rescue Measured::UnitError
 end
 ```
 
+Parse from string without having to split out the value and unit first:
+
+```ruby
+Measured::Weight.parse("123 grams")
```

Comment:
We don't show the result for all operations. And this is just showing the parser syntax.